overlord/ifacestate/handlers.go

```diff
@@ -88,12 +88,19 @@ func (m *InterfaceManager) buildConfinementOptions(st *state.State, snapInfo *sn
 		return interfaces.ConfinementOptions{}, fmt.Errorf("cannot get extra mount layouts of snap %q: %s", snapInfo.InstanceName(), err)
 	}
 
+	kernelSnap := ""
```

Comment:
the code here probably needs to take the task so we can do the right thin during remodel

Reply:
Good point, fixed now